### DISCUSSION TITLE
time in strftime format where is the document url?

### DISCUSSION THREAD

**Discussion by bronze1man** [bluenviron/mediamtx#3678] [Questions]
### Question

```
  # Available variables are %path (path name), %Y %m %d %H %M %S %f %s (time in strftime format)
```

php version do not have %f (https://www.php.net/manual/en/function.strftime.php)
python version do not have %s (https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes)

Maybe an example is better?

**Comment by aler9** [comment]
Documentation of `recordPath` has been improved with https://github.com/bluenviron/mediamtx/pull/4302
